smallrye-reactive-messaging-kafka/src/test/java/io/smallrye/reactive/messaging/kafka/MapBasedConfig.java

```diff
@@ -22,11 +22,10 @@ public MapBasedConfig(Map<String, Object> map) {
         this.map = map;
     }
 
+    @SuppressWarnings("ResultOfMethodCallIgnored")
     public static void clear() {
         File out = new File("target/test-classes/META-INF/microprofile-config.properties");
-        if (out.delete()) {
```

Comment:
Should this change to do `if (out.exists())` like the change you made in another place?

Reply:
should be `isFile`

Reply:
Does this need the check updating or ok to remove completely?

Reply:
Fixed.